OpenMarket/OpenMarket/URLSession/URLSession.swift

```diff
@@ -10,17 +10,18 @@ import Foundation
 extension URLSession: OpenMarketURLSessionProtocol {
     func fetchOpenMarketDataTask(query: String,
                                  completion: @escaping (Data?, Error?) -> Void) -> URLSessionDataTask? {
-        guard let hostURL = URL(string: host),
-              let url = URL(string: query, relativeTo: hostURL) else {
+        guard let hostURL: URL = URL(string: host),
+              let url: URL = URL(string: query, relativeTo: hostURL) else {
+            completion(nil, OpenMarketError.invalidURL(file: #file, line: #line))
```

Comment:
```suggestion
            completion(nil, OpenMarketError.invalidURL())
```
이렇게 사용부에서 사용이 될 수 있을것 같네요.

Reply:
마찬가지로 [43682a9](https://github.com/yagom-academy/ios-open-market/pull/218/commits/43682a91bf6275efd3c7f7daebb970d1a65fa61c) 에서 수정했습니다.